test/run_pass/test_ble_setservices_central.js

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2016-present Samsung Electronics Co., Ltd. and other contributors
```

Comment:
@yichoi I think 2016-present is enough. 2016 is the starting year of IoT.js project. I don't put time to update every copyright year. However, If you prefer, I will fix it.